ui/README.md

```diff
@@ -0,0 +1,14 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
I think we should remove this line.

Reply:
Will do, just wanting to give the nod to them.

Reply:
That would be the nice thing to do I suppose :-)

Reply:
+1 on removing this line, but we can do that when we refactor README I think